blog/2025/issue-credentials-over-openid4vci.adoc

```diff
@@ -0,0 +1,376 @@
+:title: Setting Up Keycloak as a Credential Issuer with OpenID4VCI
+:date: 2025-12-02
+:publish: true
+:author: Rodrick Awambeng, Forkim Enjeckayang, Ingrid Kamga, Bertrand Ogen
+:summary: Keycloak can act as a verifiable credential issuer using OpenID4VCI, allowing users to receive verifiable credentials as digital proofs of identity. This guide walks through the required configuration steps.
+
+Keycloak implements https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html[OpenID for Verifiable Credential Issuance] (OpenID4VCI). This functionality allows Keycloak to issue verifiable credentials (VCs) as a digital proof of identity or attributes. Enabling this functionality requires consistent configuration across several components, including the realm, client, and issuable credentials (as client scopes).
+
+For example, consider the following scenario: The Keycloak OAuth SIG team wants to issue verifiable membership credentials to its members, including their name and email, which can later be presented at onsite or virtual events as proof of active membership.
+
+NOTE: At the time of writing this blog, Keycloak's support for OpenID4VCI is still experimental. This guide uses the **main branch snapshot version** `999.0.0-SNAPSHOT`, and the feature is expected to be promoted to preview in early 2026.
+
+== Introduction to OpenID4VCI
+
+OpenID4VCI is a protocol developed by the OpenID Foundation that extends the OpenID Connect (OIDC) framework to enable the secure and standardized issuance of verifiable credentials (VCs). VCs are digital, tamper-evident representations of information, such as identity attributes or qualifications, that can be cryptographically verified without contacting the issuer at verification time. This promotes decentralized identity management, where users (holders) control their data and share it selectively with verifiers.
+
+++++
+<div class="paragraph" style="margin-top:1em;margin-bottom:2em;">
+    <img src="${blogImages}/openid4vci/triangle-of-trust.png" alt="Triangle of trust or Issuer-Holder-Verifier model" style="border:none;display:block;max-width:100%;height:auto;margin:0 auto;">
+    <p style="text-align:center;font-style:italic">Triangle of trust or Issuer-Holder-Verifier model</p>
+</div>
+++++
```

Comment:
Please use inline HTML only if you need specific formatting / CSS of the images. In all other cases, use the standard AsciiDoc syntax

```
image::... 
```

That even allows for titles of images 

```
.Title
image::... 
```